sass.py

```diff
@@ -556,8 +556,11 @@ def _get_file_arg(key):
     source_map_filename = _get_file_arg('source_map_filename')
     output_filename_hint = _get_file_arg('output_filename_hint')
 
-    # #208: cwd is always included in include paths
-    include_paths = (os.getcwd(),)
+    # #208: cwd is always included in include paths, if possible
+    try:
+        include_paths = (os.getcwd(),)
+    except OSError:
```

Comment:
worth calling out why this `OSError` can occur, perhaps something like: "if the current working directory has been deleted, `getcwd()` will raise"

might also be good to write a quick regression test to demonstrate this -- let me know if you'd like some help with this -- it should be pretty easy to start with something like this:

```python
def test_compile_in_missing_dir(tmpdir):
    does_not_exist = tmpdir.join('does_not_exist').ensure_dir()
    with does_not_exist.as_cwd():
        does_not_exist.remove()
        # run a compile from here
```